src/main/java/pages/CasualDressPage.java

```diff
@@ -0,0 +1,41 @@
+package pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import static pages.SignInPage.driver;
+
+public class CasualDressPage {
+  @FindBy(css = ".product-container")
+  static
+  WebElement casualdresshover;
+  @FindBy(css = ".button.ajax_add_to_cart_button.btn.btn-default")
+  static
+  WebElement addtochart;
+
+  @FindBy(css = ".col-md-6 > div.button-container > a")
+  static
+  WebElement proceedbutton;
```

Comment:
why different line for static and WebElement

Reply:
fixed in new commit